backend/app/assets/javascripts/spree/backend/admin.js

```diff
@@ -66,13 +74,9 @@ Spree.ready(function(){
       el.prop("id", el.prop("id").replace(/\d+(?=[^\d]*$)/, new_id))
       el.prop("name", el.prop("name").replace(/\d+(?=[^\d]*$)/, new_id))
     })
-    // When cloning a new row, set the href of all icons to be an empty "#"
-    // This is so that clicking on them does not perform the actions for the
-    // duplicated row
-    new_table_row.find("a").each(function () {
-      var el = $(this);
-      el.prop('href', '#');
-    })
+    // actions
```

Comment:
Maybe `Add a remove button to the actions column of the new row` can help more?

Reply:
yep. i would improve that comment also this way!